UK Games/UK Games/Controllers/UserController.cs

```diff
@@ -40,23 +30,117 @@ public IActionResult ForgotPassword()
     }
 
     [HttpGet]
-    public IActionResult Login()
+    public IActionResult Register()
     {
         return View();
     }
 
     [HttpPost]
-    public IActionResult Login(IFormCollection fc)
+    public IActionResult Register(IFormCollection fc)
+    {
+        try
+        {
+            string first = fc["FirstName"];
+            string last = fc["LastName"];
+            DateTime dob = DateTime.Parse(fc["DOB"]);
```

Comment:
DateTime.Parse throws FormatException when the  DOB is invalid. Consider adding specific error message notifying the user if their DOB is invalid e.g. user enters in the format MM/DD/YY instead of MM/DD/YYYY.

Perhaps something like 
```
            DateTime dob;
            try { dob = DateTime.Parse(fc["DOB"]); }
            catch(FormatException f) { ModelState.AddModelError("", "Date of birth format error."); return View(); }

```